src/components/grid/grid-container/grid-container.component.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import GridContainerStyle from './grid-container.style';
+
+const GridContainer = (props) => {
+  const {
+    children
+  } = props;
+  return (
+    <GridContainerStyle>
+      {children}
+    </GridContainerStyle>
+  );
+};
+
+GridContainer.propTypes = {
+  /** Defines the GridItem Components to be rendered within the GridContainer */
+  children: PropTypes.node
```

Comment:
Are `children` required, or can you use `GridContainer` without any contents? If it's required we should use `.isRequired`.

Also, consider adding a custom prop type to ensure that people only use `GridItem` within.